platforms/core-configuration/kotlin-dsl/src/main/kotlin/org/gradle/kotlin/dsl/accessors/runtime/Runtime.kt

```diff
@@ -42,7 +42,7 @@ fun conventionPluginOf(target: Any, name: String) =
 
 @Suppress("deprecation")
 fun conventionPluginByName(convention: org.gradle.api.plugins.Convention, name: String): Any =
-    convention.plugins[name] ?: throw IllegalStateException("A convention named '$name' could not be found.")
+    DeprecationLogger.whileDisabled(Factory { convention.plugins[name] }) ?: throw IllegalStateException("A convention named '$name' could not be found.")
```

Comment:
🤔 This is invoked by `AccessorsFragments#fragmentsForConvention` and I think we should not disable the deprecation logger here. We want consumers of accessors to convention to get the warning.